lib/hummingbird_survey/forms/survey_page_form.rb

```diff
@@ -148,6 +148,7 @@ def add_final_item(hash, item)
       hash["question_#{item.survey_itemable.id}"] = {
         "item_number" => item.item_number,
         "label" => item.survey_itemable.label,
+        "question_type" => item.survey_itemable.question_type,
```

Comment:
@Onikoroshi I tried to save the question type, but wasn't able to get it to stick.

Reply:
@cgratigny : The survey page form is not where you change the elements of the survey. That's in the staff area.